kafka-zookeeper/docker-bin/docker-healthcheck

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# HEALTHCHECK for use in `docker ps` or `docker-compose ps`.
+# It can also be used as a readiness probe in k8s
+
+# Fail on unset variables, but don't quit on rc!=0, so we can log what happened
```

Comment:
This is the script re-written and I'll do similarly everywhere something is a composite service or otherwise blamed.. ahem.. pinot.. ahem